File: unit2/project/variables.c
#include <stdio.h>
#include<stdlib.h>

int i;
float add=0;
float x=0;
float av=0;

int main(int argc, char* argv[]){
    
    for (i=1; i<argc; i++){ 
        x = atof(argv[i]);
        add += x;
        av = (add)/i;
    if (i==(argc-1)){
    printf("Average:%.3f\n", av);  
    }
   
    }

    return 0;
}

 

File: unit2/project/variables.py
import sys

i=0
add=0
av=0

if __name__ == "__main__":
    print(f"{sys.argv[0]}: Average of entered numbers. \n")
    for i in range (1, len(sys.argv)):
        add=(add+float(sys.argv[i]))
    av=add/i
    
    print("The average is: ", av)